recipes/pgcgap/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
   run:
     - abricate
     - canu >=1.8
-    - openjdk
+    - openjdk =8.0.152
```

Comment:
@liaochenlanruo I recall you had previously removed this version pinning in your last PR here 
https://github.com/bioconda/bioconda-recipes/pull/24313 - is it really necessary to pin it?

Reply:
Yes, It is necessary. I found that java does not work well with the latest version 8, and v8.0.152 is ok.

Reply:
Okay - noted 👍